src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  @include flex-column-centred;
+
+  max-width: $card-width;
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  @include hover-transition;
+
+  &__product-image {
+    max-width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__header {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
+  }
+
+  &__product-code-box {
+    margin-bottom: 16px;
+    align-self: start;
+  }
+
+  &__code {
+    @include zero-margin;
+    @include small-font-size;
+    color: $secondary-font-color;
+  }
+
+  &__review-box {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    width: 100%;
+  }
+
+  &__reviews {
+    @include zero-margin;
+    @include small-font-size;
+  }
+
+  &__price-box {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+    width: 100%;
+  }
+
+  &__price-text {
+    @include zero-margin;
+    @include medium-font-size;
+    color: $secondary-font-color;
+  }
+
+  &__price-value {
+    @include zero-margin;
+    @include large-font-size;
+  }
+
+  &__link {
+    width: 166px;
```

Comment:
avoid hardcoding width - unless you need it
think of some other option - just to let button fill the parent